tests/db/test_bigchain_api.py

```diff
@@ -22,52 +26,66 @@ def test_remove_unclosed_sockets():
 
 
 class TestBigchainApi(object):
+    def test_create_transaction_create(self, b, user_sk):
+        tx = b.create_transaction(b.me, user_sk, None, 'CREATE')
 
-    def test_create_transaction(self, b):
-        tx = b.create_transaction('a', 'b', 'c', 'd')
-
-        assert sorted(tx) == sorted(['id', 'transaction'])
-        assert sorted(tx['transaction']) == sorted(['current_owner', 'new_owner', 'input', 'operation',
-                                                    'timestamp', 'data'])
+        assert sorted(tx) == sorted(['id', 'transaction', 'version'])
+        assert sorted(tx['transaction']) == sorted(['conditions', 'data', 'fulfillments', 'operation', 'timestamp'])
```

Comment:
Just for the sake of simplicity: is it really needed to sort the right hand side? Especially, given that it is already sorted when handed over to `sorted()`.